### DISCUSSION TITLE
npm ERR! Class extends value undefined is not a constructor or null

### DISCUSSION THREAD

**Discussion by ycode99** [tailwindlabs/tailwindcss#13753] [Help]
![Screenshot 2024-05-28 015709](https://github.com/tailwindlabs/tailwindcss/assets/105976850/530acd24-722f-4464-b738-7c61d0b4defa)

**Comment by acewf** [comment]
doesn't seems to be related to tailwindcss, and the error isn't clear what is about. Did you try install other packages?
